backend/grant/app.py

```diff
@@ -20,12 +21,18 @@ def create_app(config_objects=["grant.settings"]):
     register_blueprints(app)
     register_shellcontext(app)
     register_commands(app)
+
     if not app.config.get("TESTING"):
         sentry_sdk.init(
             environment=ENV,
             release=SENTRY_RELEASE,
             integrations=[FlaskIntegration()]
         )
+
+    # handle all AuthExceptions thusly
+    # NOTE: testing mode does not honor this handler, and instead returns the generic 500 response
+    app.register_error_handler(AuthException, handle_auth_error)
```

Comment:
This is for all errors, right? Might we be sending exceptions back to the client that bubble up unexpectedly?

Reply:
Nope, specifically for AuthException
http://flask.pocoo.org/docs/dev/errorhandling/#registering

Reply:
What I struggled with was testing, I threw in the towel after an hour of trying to get it to use the AuthException handler instead of the default testing one.

Reply:
Interesting. We can manually test this in https://github.com/grant-project/zcash-grant-system/issues/199 (linking so it'll show up in the issue)